osquery/tests/integration/tables/deb_packages.cpp

```diff
@@ -14,30 +13,40 @@
 
 #include <osquery/tests/integration/tables/helper.h>
 
+#include <osquery/logger.h>
+
 namespace osquery {
 
-class debPackages : public IntegrationTableTest {};
-
-TEST_F(debPackages, test_sanity) {
-  // 1. Query data
-  // QueryData data = execute_query("select * from deb_packages");
-  // 2. Check size before validation
-  // ASSERT_GE(data.size(), 0ul);
-  // ASSERT_EQ(data.size(), 1ul);
-  // ASSERT_EQ(data.size(), 0ul);
-  // 3. Build validation map
-  // See IntegrationTableTest.cpp for avaialbe flags
-  // Or use custom DataCheck object
-  // ValidatatioMap row_map = {
-  //      {"name", NormalType}
-  //      {"version", NormalType}
-  //      {"source", NormalType}
-  //      {"size", IntType}
-  //      {"arch", NormalType}
-  //      {"revision", NormalType}
-  //}
-  // 4. Perform validation
-  // EXPECT_TRUE(validate_rows(data, row_map));
+class DebPackages : public IntegrationTableTest {};
+
+TEST_F(DebPackages, test_sanity) {
+  QueryData rows = execute_query("select * from deb_packages");
+  if (rows.size() > 0) {
+    ASSERT_GE(rows.size(), 4ul);
```

Comment:
Why do you assume it is 4 rows? Add comment.

Reply:
ok, make sense, thanks!